ext/openssl/ossl_cipher.c

```diff
@@ -813,6 +813,31 @@ ossl_cipher_block_size(VALUE self)
     return INT2NUM(EVP_CIPHER_CTX_block_size(ctx));
 }
 
+/*
+ *  call-seq:
+ *     cipher.ccm_data_len = integer -> integer
+ *
+ *  Sets the length of the plaintext / ciphertext message that will be
+ *  processed in CCM mode. Make sure to call this method after #key= and
+ *  #iv=fe have been set, and before #auth_data=.
```

Comment:
Typo?

Reply:
Yeah that's a typo, I'll get a commit fixing it pushed up shortly.